flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/operations/SqlToOperationConverter.java

```diff
@@ -864,6 +874,31 @@ private Operation convertDescribeTable(SqlRichDescribeTable sqlRichDescribeTable
         return new DescribeTableOperation(identifier, sqlRichDescribeTable.isExtended());
     }
 
+    /** Convert LOAD MODULE statement. */
+    private Operation convertLoadModule(SqlLoadModule sqlLoadModule) {
+        String moduleName = sqlLoadModule.moduleName();
+        Map<String, String> properties = new HashMap<>();
+        for (SqlNode node : sqlLoadModule.getPropertyList().getList()) {
+            SqlTableOption option = (SqlTableOption) node;
+            if (MODULE_TYPE.equals(option.getKeyString())) {
```

Comment:
Could you also move this check to TableEnv? This logic should appare with "type" injection.